packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -451,7 +455,7 @@ export async function stableScreenshot(
   try {
     return await page.screenshot(screenshotOptions);
   } finally {
-    if (stubOptions?.dismissTooltips) {
+    if (stubOptions?.dismissTooltips !== false) {
```

Comment:
Should 
```
  if (stubOptions?.dismissTooltips) {
    await disableTooltips(page);
  }
```

Also have the updated `!== false` check

Reply:
Eh right. I didn't actually fix the bug.